app/src/main/java/org/sopt/androidstudy/FriendLocalDataSource.kt

```diff
@@ -0,0 +1,60 @@
+package org.sopt.androidstudy
+
+import androidx.lifecycle.LiveData
+import org.sopt.androidstudy.data.models.types.MBTI
+import org.sopt.androidstudy.data.models.types.MBTIFeatures
+import org.sopt.androidstudy.db.Friend
+import org.sopt.androidstudy.db.FriendDAO
+
+class FriendLocalDataSource(val dao: FriendDAO) {
```

Comment:
```suggestion
class FriendLocalDataSource(private val dao: FriendDAO) {
```

외부에서 dao에 접근할 것이 아니라면 private를 붙여주세여~!